proto/p4/config/p4info.proto

```diff
@@ -30,6 +62,8 @@ message P4Info {
   repeated Meter meters = 7;
   repeated DirectMeter direct_meters = 8;
   repeated ControllerPacketMetadata controller_packet_metadata = 9;
+  PkgInfo pkg_info = 10; 
```

Comment:
No concerns about breaking .proto compatibility?

Reply:
We just had a problem manifest itself in a strange way due to recent P4Runtime renumbering, so I'm not going to say I don't mind.

Reply:
I'll leave numbering as-is to avoid breakage although I agree it would be nice to make PkgInfo = 1.

Reply:
If @teverman wants to avoid renumbering, maybe we should assign a large number to pkg_info (such as 1000) and keep it at the end of the message? We are still adding entries to the P4Info message (e.g. parser value sets) and it's going to look weird to have pkg_info in the middle...

Reply:
How about swapping the field numbers for Extern and PkgInfo, i.e. set Extern = 10 and PkgInfo = 1. There may be some mitigating factors favoring this approach:
* https://github.com/p4lang/p4c/issues/1164 - It's possible that externs aren't emitted in P4Info anyway so it won't affect anyone
* It's possible nobody is using externs in P4Runtime other than the ones which get converted into meters and counters.
* Even if the above two points are not true, it only impacts one field, vs. all of them if PkgInfo were to be inserted at the top instead of swapped with the top.

Reply:
I'm no expert on Google Protobufs, but aren't people not supposed to care about the numeric values much, except to make sure they are distinct?

Reply:
@jafingerhut: Various guidelines exist for updating protocol buffers to maintain client-server version compatibility.  One of them is "don't renumber fields".  If you add a new feature and renumber fields in the process, then when an old client with the old numbering scheme sends a message to the new server with the changed numbering scheme, things can fail in mysterious ways.  If the field numbers are left alone, then the old client simply doesn't have access to the server's new feature, but all the old features still work.

I am OK myself with swapping Extern and PkgInfo so the latter is first, but I vaguely remember someone insisting that Extern needed to be in front of everything else when it was added.

Reply:
I changed PkgInfo to 1000 pending further group discussion.